source/docs/software/dashboards/troubleshooting-dashboard-connectivity.rst

```diff
@@ -0,0 +1,64 @@
+Troubleshooting Dashboard Connectivity
```

Comment:
Can the sections about the different dashboards be made into tabs rather than subsequent sections?

Reply:
Tabs don't work well in all media, so we should only use them where it's really helpful (like code)